src/content/structured/components/data-entity/code.mdx

```diff
@@ -0,0 +1,548 @@
+---
+path: "/components/data-entity/code"
+
+date: "2022-11-15"
+
+title: "Data entity"
+
+status: "BETA"
+
+subTitle: "The data entity component displays a group of related data in a consistent and easy to consume way."
+
+contribute: "https://github.com/mi6/ic-design-system/tree/main/src/content/structured/components/data-entity"
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/data-entity" },
+    { title: "Code", path: "/components/data-entity/code" },
+    { title: "Accessibility", path: "/components/data-entity/accessibility" },
+  ]
+---
+
+import {
+  IcDataEntity,
+  IcDataRow,
+  IcLink,
+  IcTypography,
+  IcTextField,
+  IcButton,
+  IcStatusTag,
+} from "@ukic/react";
+
+## Component demo
+
+export const snippets = [
+  {
+    language: "Web component",
+    snippet: `<ic-data-entity heading="My details">
+  <ic-data-row label="First name" value="Dave"><ic-link href="/" slot="end-component">Edit</ic-link></ic-data-row>
```

Comment:
Syntax highlighting is off. Is it better if you move ic-link onto a new line?? Same with react snippet

![image](https://user-images.githubusercontent.com/116102637/204000656-6d53717b-4531-4321-a566-7a194c8a0930.png)